ColorCode.UWP/RichEditBoxFormatter.cs

```diff
@@ -0,0 +1,196 @@
+﻿using System.Collections.Generic;
+using ColorCode.Parsing;
+using Windows.UI.Xaml.Controls;
+using ColorCode.Styling;
+using Windows.UI.Text;
+using ColorCode.UWP.Common;
+using ColorCode.Common;
+using Windows.UI.Xaml;
+using System.Text.RegularExpressions;
+
+namespace ColorCode
+{
+    /// <summary>
+    /// Creates a <see cref="RichTextBlockFormatter"/>, for rendering Syntax Highlighted code to a RichTextBlock.
+    /// </summary>
+    public class RichEditBoxFormatter : CodeColorizerBase
```

Comment:
Seems like this could this be an extension with an attached property instead of a helper class?

Reply:
It could be both, I would like to keep the formatter class style, as that is what the other Formatters are.

I haven't played with extensions before, but I'll look into it.

Reply:
How would I set a Markup Extension for a specific control only, is there some documentation on creating UI Extensions for more info than I can garner from the ones in the Toolkit?

Reply:
You just change the type in the Get/SetValue from DependencyProperty to the specific control to limit it (like they did in [TextBoxRegEx](https://github.com/Microsoft/UWPCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.UI/Extensions/TextBoxRegEx/TextBoxRegex.Properties.cs)).  Is that what you're asking?